docs/uml/classDiagrams/exercise.puml

```diff
@@ -0,0 +1,35 @@
+@startuml
+'https://plantuml.com/class-diagram
+hide circle
+title Class Diagram of `exercise` Command
+Main -> WerkIt
+WerkIt -> Parser
+WerkIt -> UI
+"{abstract}\nCommand" <|- ExerciseCommand
+
+
+Parser ..> "{abstract}\nCommand"
+class WerkIt {
++ startContinuousUserPrompt()
+}
+
+
+class Main {
+
+}
+
+class ExerciseCommand {
++ execute()
+}
+
+class UI {
++ getUserInput()
+}
+
+class Parser {
++ parseUserInput(String userInput)
++ ExerciseCommand createExerciseCommand(String userInput)
+}
+
+
```

Comment:
Perhaps diagram might need `ExerciseList.java` class also to show the actions `Exercises` can do. Otherwise, the rest of the diagram looks good!

Reply:
The `execute` will call printExerciseList() from ExerciseList, I've updated the changes.

Reply:
very nice!